### PR TITLE
Make adjustments to the Input component

### PR DESCRIPTION
**Pull request by pinguinjkeke**

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @Dantiff, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Input/Input.component.tsx [hunk 1]
```diff
@@ -171,6 +167,11 @@ const SInputWrapper = styled.div`
       margin-left: -1px;
     }
   }
+  &.disabled, &.disabled > input {
```
Comment (Dantiff) [inline src/Input/Input.component.tsx]: Seems like setting the `invalid` prop to true does not add the invalidBorderColor

image

File: src/Input/Input.component.tsx [hunk 2]
```diff
@@ -185,19 +186,24 @@ export const Input: React.FunctionComponent<Props> = ({
...
+          )}
           {...inputProps}
         >
           {('left' === inputProps.iconPosition ||
             'left' === inputProps.addonTextPosition) && (
             <span>{inputProps.icon || inputProps.addonText}</span>
           )}
           <SInput
-            value={value}
+            value={(value || '').substring(0, inputProps.maxLength)}
```
Comment (Dantiff) [inline src/Input/Input.component.tsx]: The maxLength does not seem to work either

image

Comment (Dantiff) [reply src/Input/Input.component.tsx]: Works as expected

**Pull request merged**